src/modules/bets.js

```diff
@@ -19,10 +19,55 @@ const parse = str => {
 
 export default function bets(opts) {
 
-  function bet(bot, options) {
+  const { db } = opts
+
+  function bet(bot, options, status = 'open', _entries = {}) {
+    db.schema.createTableIfNotExists('bet_options', (table) => {
+      table.string('name').primary()
+      table.string('represents')
+    }).then(() => {
+      debug('created table bet_options')
+      db('bet_options').del().then(() =>
+        db('bet_options').insert(Object.keys(options).map(name => {
+          return { name: name, represents: options[name] }
+        }))
+      ).catch(e => {
+        throw e
+      })
+    }).catch(e => {
+      throw e
+    })
```

Comment:
This can be flattened to something like
```js
db.schema.createTableIfNotExists('bet_options', (table) => {
  table.string('name').primary()
  table.string('represents')
})
  .tap(() => debug('created table bet_options'))
  .then(() => db('bet_options').del())
  .then(() => {
    return db('bet_options').insert(Object.keys(options).map(name => {
      return { name: name, represents: options[name] }
    }))
  })
  .catch(e => { throw e })
```

(`.tap()` is like `.then`, but doesn't influence the return value)

Same below, although the bet_entries one might be a bit harder :P